pyproject.toml

```diff
@@ -20,32 +17,38 @@ classifiers = [
     "Topic :: Software Development :: Debuggers",
     "Topic :: Internet :: Log Analysis"
 ]
+requires-python = ">=3.11"
+
+dependencies = [
+    "requests>0.2.31",
+    "llama-cpp-python>0.2.56, !=0.2.86",
+    "drain3 >=0.9.11, < 1.0.0",
+    "huggingface-hub>0.23.2",
+    # rawhide has numpy 2, F40 and F41 are still on 1.26
+    # we need to support both versions
+    "numpy>=1.26.0",
+    "python-gitlab >=5.6.0, < 6.0.0",
```

Comment:
In order to build for F41, we need to relax the dependency to `"python-gitlab >=4.10.0, < 6.0.0"`